src/dll/Hooks/ValidateScripts.cpp

```diff
@@ -89,3 +89,59 @@ bool Hooks::ValidateScripts::Detach()
     isAttached = result != NO_ERROR;
     return !isAttached;
 }
+
+std::wstring WritePopupMessage(const std::vector<ValidationError>& validationErrors,
+                               const std::vector<PluginSystem::PluginName>& incompatiblePlugins)
+{
+    if (validationErrors.empty())
+    {
+        return {};
+    }
+
+    std::wstring message;
+
+    if (!incompatiblePlugins.empty())
+    {
+        fmt::format_to(std::back_inserter(message),
+                       L"The following red4ext plugins could not be loaded because they are "
```

Comment:
```suggestion
                       L"The following RED4ext plugins could not be loaded because they are "
```
or
```suggestion
                       L"The following plugins could not be loaded because they are "
```